client/src/main/java/com/chattycathy/client/handler/Message.java

```diff
@@ -0,0 +1,19 @@
+package com.chattycathy.client.handler;
+
+/**
+ * Expandable class for constructing message dataframe to transmit
+ */
+public class Message {
+    String body;
+
+    /**
+     * Instantiates Message class with message body
+     */
+    public Message(String body) {
+        this.body = body;
+    }
+
+    public String getMessage() {
+        return this.body;
+    }
+}
```

Comment:
This can be made concise with Lombok (note that you will need to add imports):
```suggestion
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * Expandable class for constructing message dataframe to transmit
 */
@AllArgsConstructor // Creates a constructor with all arguments: body
@NoArgsConstructor // Creates a constructor with no arguments
@Getter // Provides getters for all attributes
@Setter // Provides setters for all attributes
public class Message {
    private String body;
}
```

This means if you add any attributes you won't need toa dd setters and getters afterwards, making the code less human error prone as there is no checks during build for these getter and setters.

Also this file should be in the model package not in the handler. This will also be moved to common with #19

Reply:
very good suggestion, will implement this and update :)